collectors/ebpf.plugin/README.md

```diff
@@ -39,815 +35,826 @@ The following two features from the Linux kernel are used by Netdata to run eBPF
-function call, this number is small for a human perspective, but the functions are called many times creating an impact
-on host. See the [configuration](#configuration) section for details about how to enable them.
+### Configure the eBPF collector
 
-#### Processes
+You can configure the eBPF collector's behavior to fine-tune which metrics you receive and [optimize performance](#performance opimization).
 
-Internally, the Linux kernel treats both processes and threads as `tasks`. To create a thread, the kernel offers a few
-system calls: `fork(2)`, `vfork(2)`, and `clone(2)`. To generate this chart, the eBPF
-collector uses the following `tracepoints` and `kprobe`:
+To edit the `ebpf.d.conf`:
 
-- `sched/sched_process_fork`: Tracepoint called after a call for `fork (2)`, `vfork (2)` and `clone (2)`.
-- `sched/sched_process_exec`: Tracepoint called after a exec-family syscall.
-- `kprobe/kernel_clone`: This is the main [`fork()`](https://elixir.bootlin.com/linux/v5.10/source/kernel/fork.c#L2415)
-   routine since kernel `5.10.0` was released.
-- `kprobe/_do_fork`: Like `kernel_clone`, but this was the main function between kernels `4.2.0` and `5.9.16`
-- `kprobe/do_fork`: This was the main function before kernel `4.2.0`.
+1. Navigate to the [Netdata config directory](/docs/configure/nodes.md#the-netdata-config-directory).
+    ```bash
+    cd /etc/netdata
+    ```
+2. Use the [`edit-config`](/docs/configure/nodes.md#use-edit-config-to-edit-configuration-files) script to edit [`ebpf.d.conf`](https://github.com/netdata/netdata/blob/master/collectors/ebpf.plugin/ebpf.d.conf).
 
-#### Process Exit
+   ```bash
+   ./edit-config ebpf.d.conf
+   ```
+   You can now edit the behavior of the eBPF collector. The following sections describe each configuration option in detail.
 
-Ending a task requires two steps. The first is a call to the internal function `do_exit`, which notifies the operating
-system that the task is finishing its work. The second step is to release the kernel information with the internal
-function `release_task`. The difference between the two dimensions can help you discover
-[zombie processes](https://en.wikipedia.org/wiki/Zombie_process). To get the metrics, the collector uses:
+### `[global]` configuration options
 
-- `sched/sched_process_exit`: Tracepoint called after a task exits.
-- `kprobe/release_task`: This function is called when a process exits, as the kernel still needs to remove the process
-  descriptor.
+The `[global]` section defines settings for the whole eBPF collector.
 
-#### Task error
+#### eBPF load mode
 
-The functions responsible for ending tasks do not return values, so this chart contains information about failures on
-process and thread creation only.
+The collector uses two different eBPF programs. These programs rely on the same functions inside the kernel, but they
+monitor, process, and display different kinds of information.
```

Comment:
I suggest to change `process` to `function`, because task is any software or thread, while `function` is what we monitor.

Reply:
`process` is a verb in this context: "These programs [...] monitor, process and display different kinds of information".